docs/source/release-notes/2020-2-5-Asteria.md

```diff
@@ -0,0 +1,74 @@
+
+## Interface (Codename Athena)
+
+The numbers at the end of each item are the PR numbers in the Project Athena [repo](https://github.com/kasenvr/project-athena/).
+
+#### General
+
+* Application: Improve performance of entity updates. (#276)
+* Application: Allow resolving of relative URLs when importing entities with JSON. (#586, #672)
+* App: Add incremental suffix to cloned entities' names using the Create app. (#603)
+* App: Update the `colorSpread` color-picker and other misc. fixes in the Create app. (#606)
+* API: Add `About` JavaScript API to replace the now deprecated `HifiAbout` API. (#624)
+* Application: Fix freezing when opening the Avatar app when custom bookmarks are present that do not have thumbnails. (#631)
+* UI & Security: Improve the OAuth2 domain feature to not show on domains that do not have it enabled; also enable persistence of logins. (#639)
+* App: Add update notification message on Interface start if a new version is available. (#658)
+
+#### Miscellaneous
+
+* Application: Code-signing has been re-enabled on Windows. (#303)
+* Application: Improve the handling of C++ exceptions by Crashpad on Windows. (#441)
+* App: Update the URLs to the default particle and image entities for the Create app. (#604)
+* Docs: Update build docs to include parameters for building client and server only packages. (#652)
+* Application & Docs: Update the application folder icons for Vircadia on OSX; update build docs for OSX to be more up to date. (#653)
+
+## Domain Server
+
+The numbers at the end of each item are the PR numbers in the Project Athena [repo](https://github.com/kasenvr/project-athena/).
+
+#### General
+
+* Server: Improve performance of entity updates. (#276)
+* UX: New metadata fields added to improve domain details alongside the addition of an HTTP exporter that makes this data publicly available as JSON over HTTP if desired. (#465)
+* UI: Updated text references from High Fidelity -> Vircadia in the `Assignment` tab of the domain-server panel. (#641)
+
+## Metaverse Server (Codename Iamus)
+
+The metaverse server has been renamed to [Iamus](https://github.com/kasenvr/iamus), known as Apollo's son in Greek mythology. It is beginning active testing with the wider Vircadia community. The [metaverse server interface](https://github.com/kasenvr/project-iamus-dashboard) is under continued development.
+
+## Vircadia Launcher (Codename Pantheon)
+
+[Vircadia's launcher](https://github.com/vircadia/pantheon-launcher) has not had any updates since the last release (2020.2.4 Asteria).
+
+## Extra Applications
+
+#### Explore App
+
+The numbers at the end of each item are the PR numbers in the Project Athena [repo](https://github.com/kasenvr/project-athena/).
+
+* The Explore app UI is getting a complete visual and interaction overhaul, courtesy of Vircadia collaborator Madders! (#654)
```

Comment:
I wonder whether "in development / future" things should really be in the release notes - as there are no changes in the actual release. Perhaps they would be better in a "project status" thing of some sort?